### ISSUE TITLE
New feature: split columns in doing/done and add swimlanes

### ISSUE THREAD

**Issue by edtro** [edtro/EdTro.AzureDevOps.Extensions.querybasedboards#7]

**Comment by edtro** [comment]
maybe this can also be done based on boolean fields.

**Comment by Lazys** [comment]
Hi, any progress on this and others issues? We're really looking forward new features :)

**Comment by edtro** [comment]
@Lazys Hi, I just returned from my summer vacation... and I will be picking up things very soon. For the swimlines I have already very promising results, but still have some bugs to resolve. Maybe you are interested in helping me out with pre-release testing? I have setup a seperate private offer/extension on the marketplace to enable this.
Kr
Edward

**Comment by Lazys** [comment]
I see, great :) That's good idea, I'd love to help you out with testing - we're using in the company your extension on the daily basis, so we can easily verify if nothing has been broken.

Do you need our organization name or anything?

**Comment by w0nders** [comment]
@edtro Hi, I also interested for testing features.
I think, in your extension lack feature for configure result board, i mean which field can display on work item

**Comment by edtro** [comment]
@w0nders Hi, indeed this is something that is on my todo list... see #12

**Comment by edtro** [comment]
Here is a screenshot of the preview
![image](https://user-images.githubusercontent.com/27762373/97555492-787cb080-19d8-11eb-872d-543b37469af0.png)

And used this config
```json
{
   "columns":[
      {
         "name":"New",
         "title":"New/Backlog",
         "isBacklog":true
      },
      {
         "name":"Active",
         "title":"In Progress/Active",
         "doingDone":{
            "field":"Custom.IsDone",
            "doingTitle":"Working on it...",
            "doingDefaultValue":false,
            "doneTitle":"Finished!",
            "doneValue":true
         }
      },
      {
         "name":"Test",
         "title":"Testing"
      },
      {
         "name":"Resolved",
         "title":"Resolved/Ready for release",
         "isBacklog":true
      },            
      {
         "name":"Closed",
         "title":"Completed/Done",
         "isBacklog":true
      }
   ],
   "swimlanes":{
      "field":"Microsoft.VSTS.CMMI.Blocked",
      "defaultValue": "No",
      "values":[
         {
            "value":"Yes",
            "title":"Is Blocked"
         }
      ]
   }
}
```

**Comment by edtro** [comment]
Is available as PREVIEW (with v0.20305.3 that is just released).

**Comment by edtro** [comment]
Swimlane header backgroundcolor is hardcoded and apparently optimized for the dark theme
